appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/src/main/resources/application.yml

```diff
@@ -0,0 +1,34 @@
+feature-management:
+  dynamic-features:
+    ShoppingCart:
+      assigner: Microsoft.Targeting
+      variants:
+      - 
+        default: true
+        name: Big
+        configuration-reference: ShoppingCart.Big
+        assignmentParameters:
+          Audience:
+            users:
+            - 
+              Alec
+            groups: []
```

Comment:
Is the indentation correct?

Reply:
Yes, groups should be in line with users.